guides/handGrenades.md

```diff
@@ -0,0 +1,107 @@
+# Hand Grenade Guide
+
+## Role Description
+
+All operators should be expected to familiarize themselves with the use of hand grenades. This guide will cover mainly fragmentation grenades, but will briefly cover the other 2 types as well.
+
+## Specialized Equipment
+
+Completing the certification for hand grenades will unlock the usage of fragmentation grenades as well as flashbang grenades.
```

Comment:
```suggestion
Completing the certification for hand grenades will unlock the usage of fragmentation grenades, flash-bangs, and stun grenades.
```

Reply:
Common spelling is flashbang and flashbang == stun grenade.

Reply:
This should use stun grenade then, it's the official terminology